src/linear_algebra/exterior_algebra/grading.lean

```diff
@@ -57,22 +57,24 @@ begin
   { rw [alg_hom.commutes, direct_sum.algebra_map_apply], refl },
   { rw [alg_hom.map_add, ihx, ihy, ←map_add], refl },
   { obtain ⟨_, rfl⟩ := hm,
-    rw [alg_hom.map_mul, ih, lift_ι, lift_ι_apply, graded_algebra.ι_apply, direct_sum.of_mul_of],
+    rw [alg_hom.map_mul, ih, lift_ι, lift_ι_apply,
+      graded_algebra.ι_apply R M, direct_sum.of_mul_of],
     exact direct_sum.of_eq_of_graded_monoid_eq (sigma.subtype_ext (add_comm _ _) rfl) }
 end
 
 /-- The exterior algebra is graded by the powers of the submodule `(exterior_algebra.ι R).range`. -/
 instance graded_algebra :
-  graded_algebra
-    ((^) (ι R : M →ₗ[R] exterior_algebra R M).range : ℕ → submodule R (exterior_algebra R M)) :=
-graded_algebra.of_alg_hom _ (lift_ι R M)
+  graded_algebra ((^) (ι R : M →ₗ[R] exterior_algebra R M).range : ℕ → submodule R _) :=
+graded_algebra.of_alg_hom _
+  -- while not necessary, the `by apply` makes this elaborate faster
+  (by apply lift_ι R M)
   -- the proof from here onward is identical to the `tensor_algebra` case
   (begin
     ext m,
     dsimp only [linear_map.comp_apply, alg_hom.to_linear_map_apply, alg_hom.comp_apply,
       alg_hom.id_apply, lift_ι],
-    rw [lift_ι_apply, graded_algebra.ι_apply, direct_sum.coe_alg_hom_of, subtype.coe_mk],
+    rw [lift_ι_apply, graded_algebra.ι_apply R M, direct_sum.coe_alg_hom_of, subtype.coe_mk],
   end)
-  (by exact lift_ι_eq)
+  (by apply lift_ι_eq)
```

Comment:
Even with the extracted def/lemma, adding `by apply` still makes elaboration faster (from ~2s to ~1.4s). Making `R M` explicit in `lift_ι_eq` and write `lift_ι_eq R M` further cut it down to ~1.2s. (Should I do that?)

Reply:
Minor changes like that are certainly worth it to save seconds, especially since you already put the work in to find them